2022-11-15 11:09:41,511 INFO    MainThread:14868 [wandb_setup.py:_flush():76] Configure stats pid to 14868
2022-11-15 11:09:41,511 INFO    MainThread:14868 [wandb_setup.py:_flush():76] Loading settings from C:\Users\Calculos\.config\wandb\settings
2022-11-15 11:09:41,512 INFO    MainThread:14868 [wandb_setup.py:_flush():76] Loading settings from d:\UniSergio\BasketScripts\WaB\wandb\settings
2022-11-15 11:09:41,512 INFO    MainThread:14868 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'_require_service': 'True'}
2022-11-15 11:09:41,512 INFO    MainThread:14868 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-11-15 11:09:41,512 ERROR   MainThread:14868 [wandb_setup.py:_flush():76] error
Traceback (most recent call last):
  File "d:\UniSergio\BasketScripts\lib\site-packages\wandb\sdk\wandb_init.py", line 1040, in init
    wi.setup(kwargs)
  File "d:\UniSergio\BasketScripts\lib\site-packages\wandb\sdk\wandb_init.py", line 204, in setup
    init_config = parse_config(
  File "d:\UniSergio\BasketScripts\lib\site-packages\wandb\sdk\wandb_helper.py", line 13, in parse_config
    params = config_util.dict_from_config_file(params, must_exist=True)
  File "d:\UniSergio\BasketScripts\lib\site-packages\wandb\sdk\lib\config_util.py", line 99, in dict_from_config_file
    raise ConfigError("config file %s doesn't exist" % filename)
wandb.sdk.lib.config_util.ConfigError: config file learning_rate doesn't exist
2022-11-15 11:09:41,512 INFO    MainThread:14868 [wandb_init.py:_log_setup():461] Logging user logs to d:\UniSergio\BasketScripts\WaB\wandb\run-20221115_110941-7mwvubqf\logs\debug.log
2022-11-15 11:09:41,512 INFO    MainThread:14868 [wandb_init.py:_log_setup():462] Logging internal logs to d:\UniSergio\BasketScripts\WaB\wandb\run-20221115_110941-7mwvubqf\logs\debug-internal.log
2022-11-15 11:09:41,512 INFO    MainThread:14868 [wandb_init.py:_jupyter_setup():411] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000021EE4839130>
2022-11-15 11:09:41,513 INFO    MainThread:14868 [wandb_init.py:init():495] calling init triggers
2022-11-15 11:09:41,513 INFO    MainThread:14868 [wandb_init.py:init():498] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.0001, 'num_neurons': 30, 'acti_fun': 'relu', 'batch_size': 200, 'epochs': 500, 'architecture': 'OneHiddenLayer', 'dataset': '2018-19_traditional', 'loss': 0.3565, 'acc': 0.8487, 'val_loss': 0.6881, 'val_acc': 0.56}
2022-11-15 11:09:41,513 INFO    MainThread:14868 [wandb_init.py:init():548] starting backend
2022-11-15 11:09:41,513 INFO    MainThread:14868 [wandb_init.py:init():552] setting up manager
2022-11-15 11:09:41,514 INFO    MainThread:14868 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2022-11-15 11:09:41,516 INFO    MainThread:14868 [wandb_init.py:init():558] backend started and connected
2022-11-15 11:09:41,517 INFO    MainThread:14868 [wandb_run.py:_label_probe_notebook():1105] probe notebook
2022-11-15 11:09:41,517 INFO    MainThread:14868 [wandb_run.py:_label_probe_notebook():1115] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-11-15 11:09:41,517 INFO    MainThread:14868 [wandb_init.py:init():636] updated telemetry
2022-11-15 11:09:41,517 INFO    MainThread:14868 [wandb_init.py:init():667] communicating run to backend with 30 second timeout
2022-11-15 11:09:42,897 INFO    MainThread:14868 [wandb_run.py:_on_init():1982] communicating current version
2022-11-15 11:09:43,032 INFO    MainThread:14868 [wandb_run.py:_on_init():1986] got version response upgrade_message: "wandb version 0.13.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-11-15 11:09:43,033 INFO    MainThread:14868 [wandb_init.py:init():700] starting run threads in backend
2022-11-15 11:09:43,052 INFO    MainThread:14868 [wandb_run.py:_console_start():1962] atexit reg
2022-11-15 11:09:43,052 INFO    MainThread:14868 [wandb_run.py:_redirect():1820] redirect: SettingsConsole.WRAP_RAW
2022-11-15 11:09:43,052 INFO    MainThread:14868 [wandb_run.py:_redirect():1885] Wrapping output streams.
2022-11-15 11:09:43,052 INFO    MainThread:14868 [wandb_run.py:_redirect():1907] Redirects installed.
2022-11-15 11:09:43,052 INFO    MainThread:14868 [wandb_init.py:init():732] run started, returning control to user process
2022-11-15 11:09:52,971 INFO    MainThread:14868 [wandb_run.py:_finish():1728] finishing run progettobasket/Integration/7mwvubqf
2022-11-15 11:09:52,972 ERROR   MainThread:14868 [jupyter.py:save_history():439] Run pip install nbformat to save notebook history
2022-11-15 11:09:52,972 INFO    MainThread:14868 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-11-15 11:09:52,972 INFO    MainThread:14868 [wandb_init.py:_jupyter_teardown():393] cleaning up jupyter logic
2022-11-15 11:09:52,972 INFO    MainThread:14868 [wandb_run.py:_atexit_cleanup():1931] got exitcode: 0
2022-11-15 11:09:52,972 INFO    MainThread:14868 [wandb_run.py:_restore():1914] restore
2022-11-15 11:09:52,972 INFO    MainThread:14868 [wandb_run.py:_restore():1920] restore done
2022-11-15 11:09:53,686 INFO    MainThread:14868 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 510
  total_bytes: 510
}

2022-11-15 11:09:53,980 INFO    MainThread:14868 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 510
  total_bytes: 510
}

2022-11-15 11:09:54,095 INFO    MainThread:14868 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 510
  total_bytes: 3867
}

2022-11-15 11:09:54,205 INFO    MainThread:14868 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 510
  total_bytes: 3867
}

2022-11-15 11:09:54,309 INFO    MainThread:14868 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3867
  total_bytes: 3867
}

2022-11-15 11:09:54,422 INFO    MainThread:14868 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3867
  total_bytes: 3867
}

2022-11-15 11:09:54,525 INFO    MainThread:14868 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3867
  total_bytes: 3867
}

2022-11-15 11:09:54,633 INFO    MainThread:14868 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3867
  total_bytes: 3867
}

2022-11-15 11:09:54,743 INFO    MainThread:14868 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3867
  total_bytes: 3867
}

2022-11-15 11:09:55,548 INFO    MainThread:14868 [wandb_run.py:_on_finish():2221] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3867
  total_bytes: 3867
}

2022-11-15 11:09:56,053 INFO    MainThread:14868 [wandb_run.py:_on_finish():2221] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 3867
  total_bytes: 3867
}
local_info {
}

2022-11-15 11:09:57,174 INFO    MainThread:14868 [wandb_run.py:_footer_history_summary_info():3305] rendering history
2022-11-15 11:09:57,174 INFO    MainThread:14868 [wandb_run.py:_footer_history_summary_info():3337] rendering summary
2022-11-15 11:09:57,179 INFO    MainThread:14868 [wandb_run.py:_footer_sync_info():3261] logging synced files
